config/locales/en/views.yml

```diff
@@ -35,6 +35,12 @@ en:
     snippet:
         characteristic_needs_feature: 'Knowledge Map characteristics can only be associated from a feature.'
         feature:
+            browse_related:
+              one: 'You can browse this subordinate place as well as its superordinate places with the tree below.'
+              other: 'You can browse those subordinate places as well as its superordinate places with the tree below.'
+            browse_summary:
+              one: 'which is presented here.'
+              other: 'which are presented here.'
```

Comment:
Better to have only: 1) words, 2) self-contained phrases or 3) sentences, but not half a sentence here BUT if this is exactly the same as what is committed in places, it should not be in both, but only in kmaps_engine. Messages should only be put once. If they belong to kmaps_engine functionality put there (unless they are specific to subjects and places).